src/backend/joanie/tests/core/test_api_organizations_course_product_relations.py

```diff
@@ -0,0 +1,85 @@
+"""
+Test suite for Organization's CourseProductRelation API endpoint.
+"""
+
+from joanie.core import factories
+from joanie.tests.base import BaseAPITestCase
+
+
+class OrganizationCourseProductRelationApiTest(BaseAPITestCase):
+    """
+    Test suite for Organization's CourseProductRelation API endpoint.
+    """
+
```

Comment:
Can you add all test to ensure that read operations are not allowed (POST, PUT, PATCH and DELETE). Take a look here : 
https://github.com/openfun/joanie/blob/96dcd4bdc462f78b2a4a00d60c264b1954fca38e/src/backend/joanie/tests/core/test_api_course_product_relations.py